core/domain/acl_decorators.py

```diff
@@ -126,3 +167,320 @@ def test_can_view_stats(self, exploration_id, **kwargs):
             raise self.PageNotFoundException
 
     return test_can_view_stats
+
+
+def can_create_exploration(handler):
+    """Decorator to check whether the user can create an exploration."""
+
+    def test_can_create(self, **kwargs):
+        if (
+                (role_services.ACTION_CREATE_EXPLORATION in self.actions) and
+                user_services.has_fully_registered(self.user_id)):
```

Comment:
The has_fully_registered returns this:
```
    return user_settings.username and user_settings.last_agreed_to_terms and (
        user_settings.last_agreed_to_terms >=
        feconf.REGISTRATION_PAGE_LAST_UPDATED_UTC)
```
Does feconf.REGISTRATION_PAGE_LAST_UPDATED_UTC gets changes? If yes, how can we be sure about that a user with non-None id is fully registered?